language-support/ts/daml-react/BUILD.bazel

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2020 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@os_info//:os_info.bzl", "is_windows")
+load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("//language-support/ts:eslint.bzl", "eslint_test")
+load("//language-support/ts:jest.bzl", "jest_test")
+load("@sdk_version//:sdk_version.bzl", "sdk_version")
+
+ts_library(
+    name = "daml-react",
+    srcs = glob([
+        "**/*.ts",
+        "**/*.tsx",
+    ]),
+    data = [
+        ":LICENSE",
+        ":package.json",
+    ],
+    module_name = "@daml/react",
+    node_modules = "@language_support_ts_deps//:node_modules",
+    tsconfig = ":tsconfig.json",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//language-support/ts/daml-ledger",
+        "//language-support/ts/daml-types",
+        "@language_support_ts_deps//:node_modules",
+    ],
+) if not is_windows else None
+
+genrule(
```

Comment:
I guess if you just reference `//:LICENSE` in `data` it ends up in the wrong place? would be good to document this in a comment.

Reply:
yes, is the same in all the ts libraries :panda_face: . I'll put a comment.